doc/install.md

```diff
@@ -1,11 +1,15 @@
 Installation
 ============
 
+Requirements
+------------
+
+Melody depends on [composer](https://getcomposer.org/) to install packages. To do so, it'll need to find an available instance of composer on your machine. It will search for `composer` and `composer.phar` in the current directory as well as in your path.
+
 Locally
 -------
 
-Download the [melody.phar](http://get.sensiolabs.org/melody.phar) file and store
-it somewhere on your computer.
+Download the [melody.phar](http://get.sensiolabs.org/melody.phar) file and store it somewhere on your computer.
```

Comment:
I always wrap text at 80 columns on purpose. Could you revert this change and also wrap the new chapter you added ? Thanks